proto/raft_serverpb.proto

```diff
@@ -117,10 +117,18 @@ enum ExtraMessageType {
     MsgHibernateRequest = 4;
     MsgHibernateResponse = 5;
     MsgRejectRaftLogCausedByMemoryUsage = 6;
+    MsgCollectPeerApplyIndexProgressRequest = 7;
+    MsgCollectPeerApplyIndexProgressResponse = 8;
+}
+
+message ExtraState {
+    bool miss_data = 1;
+    uint64 conf_ver = 2;
 }
 
 message ExtraMessage {
     ExtraMessageType type = 1;
     uint64 premerge_commit = 2;
-    repeated metapb.Peer check_peers = 3;
+    ExtraState extra_state = 3;
+    repeated metapb.Peer check_peers = 4;
```

Comment:
no need to consider backward capatibility?

Reply:
Should consider, thanks!